Composer/packages/client/src/components/Pagination/index.tsx

```diff
@@ -16,52 +17,52 @@ export interface IPaginationProps {
 }
 
 const createDropdownOption = (pageCount: number) => {
-  const options: IDropdownOption[] = [];
-  for (let i = 1; i <= pageCount; i++) {
-    options.push({ key: 'page' + i, text: '' + i });
-  }
-  return options;
+  return range(pageCount).map((_, i) => ({
+    key: `page ${i}`,
+    text: `${i}`,
+  }));
 };
 
 export const Pagination: React.FC<IPaginationProps> = (props) => {
-  const [index, setIndex] = useState(0);
+  const [index, setIndex] = useState<number>(0);
```

Comment:
Is number not inferred by the initial argument?

Reply:
Yeah, it is. I don't know why I felt compelled to change this either.